phpunit.xml.dist

```diff
@@ -1,28 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/7.1/phpunit.xsd"
-         backupGlobals="false"
-         colors="false"
-         bootstrap="tests/Application/app/autoload.php"
-         beStrictAboutTestsThatDoNotTestAnything="true"
-         stopOnError="true"
-         stopOnFailure="true"
-         stopOnIncomplete="true"
-         stopOnSkipped="true"
-         stopOnRisky="true"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/8.0/phpunit.xsd"
+         bootstrap="tests/Application/config/bootstrap.php"
+         colors="true"
 >
     <php>
         <ini name="error_reporting" value="-1" />
-        <server name="KERNEL_DIR" value="tests/Application/app/" />
+        <server name="KERNEL_DIR" value="tests/Application/" />
         <server name="IS_DOCTRINE_ORM_SUPPORTED" value="1"/>
-        <server name="KERNEL_CLASS" value="AppKernel" />
+        <server name="KERNEL_CLASS" value="Tests\SitemapPlugin\Application\Kernel" />
     </php>
 
     <testsuites>
         <testsuite name="tests">
             <directory>tests</directory>
+            <!-- Excluded these tests because they probably will be obsolete in v2 -->
```

Comment:
Not so happy with this, as it would be great to keep them to move gradually to the new set-up (generated sitemaps), but for now it might be the best option as we discussed on Slack.